Source/scrollrt.cpp

```diff
@@ -225,6 +256,68 @@ static void scrollrt_draw_cursor_item()
 	} else {
 		CelClippedDrawSafe(mx + SCREEN_X, my + cursH + SCREEN_Y - 1, pCursCels, pcurs, cursW);
 	}
+
+	// copy cursor from back buffer to a separate bitmap

```

Comment:
Can we render to a separate buffer to begin with?

Reply:
I did try that approach but the blitting functions depend on the global state and stuff like BUFFER_WIDTH too much.

Reply:
Fair

Reply:
#964 got merged so this can now be simplified by rendering to a separate buffer to begin with.

Reply:
114 comments later, I don't feel like torturing myself with this PR any second longer

Reply:
np, perfect is the enemy of the good etc; this can be done later in a separate PR